adapters/Unity/UnityAdapter/GADMAdapterUnityBannerAd.m

```diff
@@ -71,6 +71,18 @@ - (void)loadBannerWithSize:(GADAdSize)adSize {
     return;
   }
 
+  GADAdSize supportedSize = [self supportedAdSizeFromRequestedSize:adSize];
+  if (!IsGADAdSizeValid(supportedSize)) {
+    NSString *errorMsg = [NSString
+        stringWithFormat:
+            @"UnityAds supported banner sizes are not a good fit for the requested size: %@",
+            NSStringFromGADAdSize(adSize)];
+    NSError *error =
+        GADMAdapterUnityErrorWithCodeAndDescription(GADMAdapterUnityErrorSizeMismatch, errorMsg);
+    [strongConnector adapter:strongAdapter didFailAd:error];
```

Comment:
Do you need to check that the strongConnector is still valid or not or is that guaranteed at this point already?

Reply:
we did the check in line 61 within this method. so if strongConnect is not available, it should returned already at this point.